### DISCUSSION TITLE
Preview Image + Crop

### DISCUSSION THREAD

**Discussion by chriskolenko** [rpldy/react-uploady#253] [Q&A]
Hi all, 

We want to show a Preview Image of the Cropped Image. 

However, the withRequestPreSendUpdate isn't triggering any of the useEffects or updating the items we currently have received from the different events.

Just wanted to know if anyone has a solution for this.

**Comment by yoavniran** [comment]
Hey @chriskolenko 
I'd love to assist with this one. 

First thing though, did you see the way I implemented a similar flow in this [codesandbox](https://codesandbox.io/s/react-uploady-crop-and-upload-uyl4e?file=/src/App.js)?

If yes, and you still have questions, please elaborate on how you're trying to show the cropped preview.

**Reply by chriskolenko** [reply]
In that example the final preview is the original image not the cropped section of the image.

**Reply by yoavniran** [reply]
@chriskolenko 
Sure. But that's a matter of implementation.
I updated the sandbox to show the cropped image instead of the full one
